content/blog/2022-06-21-On-Boarding-Experience-PawBud.md

```diff
@@ -0,0 +1,49 @@
+
+
+### On Boarding Experience with XSF & Community Bonding Period
+Well, I must say, this is unlike any technical intern that you will experience, the level of attention and support that you will get from the organization admins and your project mentors from Day 1 is genuinely surprising. I mean I started having weekly calls with my mentor before the coding period even began, and
+I am still quite thankful that my mentor(s) spared their time to guide me. Believe it or not, as soon as the results were announced and I saw my name, my first reaction was "oops, well this is bad". I wasn't ready for it!! Meeting JC(my primary mentor) definitely helped me a lot during the community bonding period. 
+Turns out not everyone does feel the same(thanks to patiga, I knew about this), but from my past experience in internships, I do believe that self-doubt is something that our present day society profits from, and it is something that one can't afford if they want to improve.
+
+I wanted to spend as much time with the source code as I could but my university's exams did not allow me to do so. I managed to work on some minor issues though, but I still feel that I need to put in as much work as I can. As of writing this blog, it's the second week of GSoC. I have to work upon designing and
+implementing a UI for the jingle call modal and will hopefully be done with that and the tests, by the end of this week.
+
+
+### Why Choose GSoC?
+Ok, This is an interesting one, I still remember the special treatment I recieved from the whole community once the GSoC results were announced. I think Eddie instantly shared the results with the community through the XSF GSoC group chat, twitter and a couple of other XSF social media pages. I genuinely feel pampered
+by the overwhelming support by the community, but I am so proud that I chose GSoC over a usual company internship.
+
+I took a course on software engineering during my sophomore year, and until now I have never ever seen/experienced it's concepts being implemented. In Converse, I learnt more about the appliations of test-driven software development in 2 weeks than I ever did during my whole semester. Turns out, practicing a technique 
+is actually the best way to learn it. The fact that my mentor, takes out his time which he could very easily spend on developing converse just to guide me through the smallest of obstructions is what really makes this whole thing special. No manager or supervisor in any company is going to give you this kind of attention!
+Period.
+
+
+### Why I chose XSF?
+There is this book called Zucked by Roger McNamee. For those of you who haven't read it here is a gist:- Roger was one of the initial investors of Facebook, and a mentor to Mark Zuckerberg. Throughout the book, he explains how Facebook turned into an evil giant, and how the guilt he feels that he could not stop zuckerberg 
```

Comment:
You are starting a listing here, but with one entry. I think it is not required.